<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainUI</class>
 <widget class="QWidget" name="mainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>33</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Count Reaction Facebook</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Toolbox</string>
   </property>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>84</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>URL or Path:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linkText">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>611</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="getButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>GET</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="displayTable">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>691</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="statusLable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>771</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>811</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
